app/forms/referee_interface/confidentiality_form.rb

```diff
@@ -0,0 +1,26 @@
+module RefereeInterface
+  class ConfidentialityForm
+    include ActiveModel::Model
+
+    attr_accessor :confidential
+
+    validates :confidential, presence: true
+
+    def self.build_from_reference(reference:)
+      confidential = reference.confidential ? true : false
```

Comment:
```suggestion
      confidential = reference.confidential?
```

Reply:
Nice little improvement, thanks! 

https://github.com/DFE-Digital/apply-for-teacher-training/pull/10057/commits/6206c64e68e38e9ba9b2ef4fa77ece3b31694f63